.pre-commit-config.yaml

```diff
@@ -1,9 +1,4 @@
 repos:
-  - repo: https://github.com/pre-commit/mirrors-eslint
```

Comment:
It runs faster if I just run it with pnpm and the diff. It's too big to scan the whole thing every time.